cli/internal/cmd/create.go

```diff
@@ -162,21 +165,39 @@ func (c *createCmd) create(cmd *cobra.Command, creator cloudCreator, fileHandler
 		TFLogLevel:  flags.tfLogLevel,
 		TFWorkspace: constants.TerraformWorkingDir,
 	}
-	idFile, err := creator.Create(cmd.Context(), opts)
+	infraState, err := creator.Create(cmd.Context(), opts)
 	spinner.Stop()
 	if err != nil {
 		return translateCreateErrors(cmd, c.pf, err)
 	}
 	c.log.Debugf("Successfully created the cloud resources for the cluster")
 
+	idFile := convertToIDFile(infraState, provider)
 	if err := fileHandler.WriteJSON(constants.ClusterIDsFilename, idFile, file.OptNone); err != nil {
 		return err
 	}
+	state := state.State{
+		Version:        "v1",
```

Comment:
This should be a constant.
See the version number of the config file for reference